src/m365/spo/commands/file/FileSharingLinkUtil.ts

```diff
@@ -0,0 +1,33 @@
+import request, { CliRequestOptions } from "../../../../request";
+import { formatting } from "../../../../utils/formatting";
+import { urlUtil } from "../../../../utils/urlUtil";
+import { GraphFileDetails } from "./GraphFileDetails";
```

Comment:
Instead of using this custom type, could we use the typing from the Graph typings instead?

Reply:
I don't think so.
The options in this interface are retrieved from the SP REST API. We use this API request to get the Graph list ID and Graph file ID to perform a Graph request to remove sharing links. So we use this interface for a SP REST request and not a Graph request.

Reply:
I got confused by the Vroom properties, assuming that we're calling the Vroom API which is mapped to Graph. Let's keep it as-is.